### DISCUSSION TITLE
Cannot get taskcafe.toml config to work

### DISCUSSION THREAD

**Discussion by dnwjn** [JordanKnott/taskcafe#107] [Q&A] [answered]
Hey!

Since I'm not seeing full support for all config items through env variables (I want to configure email) I'm trying to get it to work with the `conf/taskcafe.toml` file.

I mount the config file as follows:
```
volumes:
      - ./conf/taskcafe.toml:/conf/taskcafe.toml
```

Unfortunately, it doesn't see or load the config file. Postgres reports errors during authentication:
```
postgres_1  | 2021-02-17 19:47:59.774 UTC [29] FATAL:  password authentication failed for user "taskcafe"
postgres_1  | 2021-02-17 19:47:59.774 UTC [29] DETAIL:  Password does not match for user "taskcafe".
postgres_1  | 	Connection matched pg_hba.conf line 95: "host all all all md5"
```

How can I fix this? Or are all the config parameters supported through env variables?

Thanks by the way for creating this, am looking forward to putting it to good use!!

**Comment by JordanKnott** [comment, marked as answer]
All config variables can be set either through the config file or through env variables. 

However if you still want to use the config file then I'm pretty sure you need to to mount it to this path: `/root/conf/taskcafe.toml`

**Reply by dnwjn** [reply]
Ah yes I see it now! I tried that before but I guess I did something wrong there... Works perfectly now, thanks for the very quick support!

**Comment by JonLittleIT** [comment]
If anyone can help explain in details on what part of the docker compose file to add the new mount it would be very helpful.

where is the mount added and how?

`/conf/taskcafe.toml`

**Reply by dnwjn** [reply]
@JonLittleIT It's been a while for me, but you should be able to add the following line to the docker-compose config under `volumes`:

`- ./conf/taskcafe.toml:/root/conf/taskcafe.toml`

But like @JordanKnott said, you can also set the values using environment variables.

**Reply by JonLittleIT** [reply]
Thx I got it working with env vars!
